### DISCUSSION TITLE
Hiddify IPA for iOS

### DISCUSSION THREAD

**Discussion by chelaxian** [hiddify/hiddify-app#1135] [Q&A] [answered]
Build from source and do it, it will only make things complicated for Hiddify team and increases number of issues people asking for help about this.
Such processes are not for ordinary users and more for advanced ones.

_Originally posted by @amirsaam in https://github.com/hiddify/hiddify-next/issues/1133#issuecomment-2257886196_
            
            At 1st i wanna say many thanks to all of authors and contributors of Hiddify project! It is hard times for many countries and its citizens having no freedom in internet. 
            2nd - Sorry for reopening new issue, but i have no other way to communicate with you. There is still no answer to our main question:
            
            "If you restricted access to iOS version due to Apple Rules only to 200 Testflights testers, then Why you decided to share IPA for 1.1.1 and 1.3.3dev but not share IPA for other last versions? It is very missleading and frustrating to see "New Release" Tab for iOS and then see "Page Not Found" when clicking on that... Please give us logical answer why you can't (or don't want) share IPA for last versions of Hiddify? 1.3.3 is buggy and 1.1.1 have no Warp button and other cool stuff. We bagging you to share any way to get and install actual version if Hiddify IPA. Thanks for you time, attention, and probably answer."

**Comment by chelaxian** [comment]
At 1st i wanna say many thanks to all of authors and contributors of Hiddify project! It is hard times for many countries and its citizens having no freedom in internet. 
2nd - Sorry for reopening new issue, but i have no other way to communicate with you. There is still no answer to our main question:
        
"If you restricted access to iOS version due to Apple Rules only to 200 Testflights testers, then Why you decided to share IPA for 1.1.1 and 1.3.3dev but not share IPA for other last versions? It is very missleading and frustrating to see "New Release" Tab for iOS and then see "Page Not Found" when clicking on that... Please give us logical answer why you can't (or don't want) share IPA for last versions of Hiddify? Or just delete "Release Tabs" for iOS and don't create them untill you mind that it can be fully publicly shared. 1.3.3 is buggy and 1.1.1 have no Warp button and other cool stuff. We bagging you to share any way to get and install actual version of Hiddify IPA. Thanks for you time, attention, and probably answer."

**Comment by amirsaam** [comment]
> If you restricted access to iOS version due to Apple Rules only to 200 Testflights testers

The limit is 10k users, but as we release new versions we need space for more users so it's up to management level how many slot be opened on each release.

> then Why you decided to share IPA for 1.1.1 and 1.3.3dev but not share IPA for other last versions?

Personally never agreed to share IPA because I knew it will never work as how it's bundled but as somehow those versions got built in release section and I sent my concern about it. That's why it never got shared again.

> It is very missleading and frustrating to see "New Release" Tab for iOS and then see "Page Not Found" when clicking on that...

I'll ask to remove that.

> Please give us logical answer why you can't (or don't want) share IPA for last versions of Hiddify? Or just delete "Release Tabs" for iOS and don't create them untill you mind that it can be fully publicly shared.

Answered already, it's not about "we don't want to share", it is an open source project the code is there you can build it yourself and do whatever you want, but we cannot support about how to install it via TrollStore, Sideloadly, AltStore and etc.

> We bagging you to share any way to get and install actual version of Hiddify IPA. 

Build from source.
The solution you mentioned won't work at all because after version 1.4 (I think) the BundleID and Bundle Group changed and the network extension for 1.3 or 1.1 are for a different BundleID and not same group with afterward releases.

**Reply by chelaxian** [reply]
I understand your point. Anyway thanks for complete answer and agreement to fix missleading Release Tabs.
Btw - I can't build from source, beacause having iPhone/iPad doesn't mean having Mac too.
I tried to build it by SparkCode and alternatives that can build some project directly on jailbroken iPhone/iPad but no luck because your project uses Flutter for iOS. 

So if you have no script that can build flutter directly in github actions, then i will search for ways to build it on windows.

Will be glad if you can give me right direction. Now i think about:
- Codemagic
- AppCircle
- Flutter Build Service in Flutter SDK for Windows

**Reply by amirsaam** [reply, marked as answer]
check hiddify's own github action

**Comment by amirsaam** [comment]
@hiddify-com please remove the IPA section from releases.

**Comment by chelaxian** [comment]
Many thanks for finaly sharing it! You are the best!
